packages/braid-design-system/src/lib/components/Tag/Tag.tsx

```diff
@@ -47,27 +57,31 @@ export const Tag = ({
         minWidth={0}
         alignItems="center"
         background="neutralLight"
-        paddingY={onClear ? undefined : 'xxsmall'}
-        paddingLeft={icon ? 'xsmall' : 'small'}
-        paddingRight={onClear ? undefined : 'small'}
+        paddingY="xxsmall"
+        paddingX={paddingXForSize[size]}
+        paddingRight={onClear ? 'xsmall' : paddingXForSize[size]}
         borderRadius="full"
       >
         <Box minWidth={0} title={children}>
-          <Text baseline={false} maxLines={1} icon={icon}>
-            {children}
+          <Text size={size} baseline={false} maxLines={1}>
+            {icon} {children}
           </Text>
```

Comment:
What's the reason to not use the `icon` slot here?

Reply:
We decided the slot layout was not necessary and resulted in the content feeling too loose particularly inside smaller versions. Opting for the inline approach which ensure better relationship between icon and label particularly when the clear button is shown.